src/main/java/com/turkcell/staj/entities/Review.java

```diff
@@ -0,0 +1,32 @@
+package com.turkcell.staj.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "reviews")
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+    @Column(name = "package_id")
```

Comment:
`@Column(name = "package_id")
private int packageId;`

bu satır FK ilişkilerinin kurulabilmesi için, aşağıdaki kod bloğuyla değiştirilmelidir.
`@ManyToOne()
@JoinColumn(name=package_id)
Package package;`